### PR TITLE
recaptcha fix + resend sms

### PR DESCRIPTION
**Pull request by levil664**

### CODE REVIEW

File: apps/schools/domains/user/handlers/auth/register.ts [hunk 1]
```diff
@@ -1,7 +1,23 @@
 import { message } from 'antd'
 import Router from 'next/router'
+import { normalizePhone } from '../../../common/utils/phone'
 
-export async function tokenHandler (phone: string, token: string, nextUrl: string, registration: any, onFinish: () => void) {
+export async function tokenHandler (token: string, form: any, nextUrl: string, registration: any, onFinish: () => void) {
```
Comment (mzavarka) [inline apps/schools/domains/user/handlers/auth/register.ts]: recaptchToken
formComponent
registrationMutation

try to find type for this variables

File: apps/schools/domains/user/handlers/auth/register.ts [hunk 2]
```diff
@@ -31,3 +47,11 @@ export async function registrationHandler (phone: string, password: string, user
         message.error('Error registration')
     }
 }
+
+export async function resendOtpHandler (recaptcha: string, resendOtp: any, onReset: () => void) {
```
Comment (mzavarka) [inline apps/schools/domains/user/handlers/auth/register.ts]: recaptchaToken
resendOtpMutation

File: apps/schools/domains/user/handlers/auth/register.ts [hunk 3]
```diff
@@ -31,3 +47,11 @@ export async function registrationHandler (phone: string, password: string, user
...
+    let id = localStorage.getItem('token')
+    let response = await resendOtp({ resend: { recaptcha: recaptcha }, id: id })
+    if (response.error?.status === 400 || response.error?.status === 401) {
+        onReset()
```
Comment (mzavarka) [inline apps/schools/domains/user/handlers/auth/register.ts]: why onReset runs only there?

File: apps/schools/domains/user/components/auth/SharedForms/ValidatePhoneForm.tsx [hunk 4]
```diff
@@ -72,6 +77,7 @@ export const ValidatePhoneForm: React.FC<IValidatePhoneFormProps> = ({
         //     }).catch(error => {
         //         console.error(error)
         //     })
+        // resendOtpHandler(token, resend, onReset)
     }, [form/*, handleReCaptchaVerify*/])
```
Comment (mzavarka) [inline apps/schools/domains/user/components/auth/SharedForms/ValidatePhoneForm.tsx]: can we delete this callback?

File: apps/schools/domains/user/handlers/auth/register.ts
Comment (mzavarka) [inline apps/schools/domains/user/handlers/auth/register.ts:1]: write documentations for functions 
it will be helpful in fututre

**Review by mzavarka — CHANGES_REQUESTED** [review]

File: apps/schools/domains/user/handlers/auth/register.ts (hunk 3 above)
Comment (levil664) [reply apps/schools/domains/user/handlers/auth/register.ts]: 400 | Probably incorrect recaptcha.
401 | Token is not verified or it is overdue.

I think these are very rare errors that can occur. They can be easily handled like this.

**Review by mzavarka — APPROVED** [review]

**Pull request merged**